src/client/components/Dropdown/DropDown.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './DropDown.css';
+
+export function DropDown({ title }) {
+  return (
+    <select>
+      <option selected value="class">
+        {title}
+      </option>
+    </select>
+  );
+}
+export function StudyGroup({ title }) {
```

Comment:
`StudyGroup` and `Skills` should use `Dropdown` for implementation, not be implemented like this.